cli/dts/lib.deno.unstable.d.ts

```diff
@@ -67,6 +67,18 @@ declare namespace Deno {
    */
   export function osRelease(): string;
 
+  /** **Unstable** new API. yet to be vetted.
+   *
+   * Set the exit code to use when Deno exits naturally or when
+   * `Deno.exit()` is called without an argument.
+   *
+   * ```ts
+   * Deno.setExitCode(42);
+   * Deno.exit(); // Exits with status code 42.
+   * ```
+   */
+  export function setExitCode(code: number): void;
```

Comment:
I'm not yet convinced we want to expose it as `Deno` API; maybe let's go with "hidden op" approach we discussed last time? Ie. there's an op to set it, but we don't expose it as `Deno` API right now.

Reply:
It doesn't really matter when it's behind --unstable, I think?

At any rate, I can't seem to call `Deno.core.opSync("op_set_exit_code", 42)` from std/node. It doesn't show up in `Deno.core.ops()` (whereas e.g. `op_exit` does) and when I invoke it, I get this:
```
Uncaught TypeError: invalid op id: expected type `v8::data::Integer`, got `v8::data::Value`
````
(Which is a terribly misleading error message, by the way.)

Reply:
Okay, belay that - it works after a full rebuild. I'll change it if you feel strongly but :shrug: